pypit/par/pypitpar.py

```diff
@@ -0,0 +1,2486 @@
+"""
+Defines parameter sets used to set the behavior for core pypit
+functionality.
+
+There are two main parameter set groups, those used for reducing the
+data:
+
+    - RunPar: Parameters specific to a given execution of PypIt
+
+    - ReducePar: Parameters the define how PypIt should perform the
```

Comment:
I like this spelling of PypIt.

We should settle on one.

Reply:
:)  I started doing it that way to see what response I got.  Then I went back to Pypit, but I like PypIt, too...